packages/node_modules/@webex/webex-core/src/config.js

```diff
@@ -47,6 +47,14 @@ export default {
      */
     validateDomains: true,
 
+    /**
+     * services does not need auth validation
```

Comment:
typo: services that don't need auth validation

Reply:
done